Kp: 200
Kd: 20
limit1: 5000
limit2: 5000
limit3: 5000
limit4: 500
limit5: 500
limit6: 500
velocity_limit1: 1
velocity_limit2: 1
velocity_limit3: 1
velocity_limit4: 1
velocity_limit5: 1
velocity_limit6: 1
dt: 0.01
weight_cost_u: 0.001
weight_cost_a: 1
weight_cost_slack: 1
w: 6